### DISCUSSION TITLE
New Interface: List page idea for mobile view

### DISCUSSION THREAD

**Discussion by gautamsi** [keystonejs/keystone#5310] [Feature requests]
I am thinking if we can also improve the list page with mobile view, currently it has to scroll left-right to see all the fields

ideas

1. Add a `Card` view for row, so that when the page is less than certain width (mobile view) or with toggle, we can switch from row to card list view. having column selected will be helpful

2. long hold should select the card and also subsequent selection would select other cards with a checkbox in the top-right or -top-left corner

3. Tapping card can open the item view as usual

@JedWatson

**Comment by bladey** [comment]
Thanks for the suggestions @gautamsi, we're aiming to improve the responsiveness of the Admin UI in the near future, we'll keep this in mind.
